### DISCUSSION TITLE
[BUG] where the command line versioning is different or missing from the web extension versioning

### DISCUSSION THREAD

**Discussion by calvin-li-developer** [dani-garcia/vaultwarden#3412] [Q&A] [answered]
### Deployment environment

* vaultwarden version: 1.28.1

* Install method: docker via `docker compose up -d`

* Clients used:  web browser extension firefox / edge /chrome

* Reverse proxy and version:  yes via cloudflare tunnels

* MySQL/MariaDB or PostgreSQL version:  n/a

* Other relevant details:

### Steps to reproduce

So I have updated to the latest version of vaultwarden using `docker compose pull` and `docker compose up -d` then I checked the version via cli with command `docker exec vaultwarden /vaultwarden --version` and got version 1.28.1

### Expected behaviour

Should show version 1.28.1 for server version on the extension about page.
### Actual behaviour
when I checked on firefox extension it still shows 1.28.0 but also when I checked via chrome or edge extension it doesnt display any version after the like server version (thrid party):
Firefox about page:
![firefox extension](https://user-images.githubusercontent.com/65045619/229397584-9933f145-de98-4271-9571-2f00e7c075d0.PNG)
Edge/Chrome about page:
![edge-chrome extension](https://user-images.githubusercontent.com/65045619/229397599-1b743d7d-df70-4145-8055-b6a5f721aedd.PNG)

**Comment by BlackDex** [comment, marked as answer]
It's probably cached on the client side.
You can check the version we return by going to `https://your.domsin.tld/api/config`

**Comment by calvin-li-developer** [comment]
Oddly enough when I reinstall bitwarden extension on firefox it has the same screenshot result has the edge / chrome one. 

> It's probably cached on the client side. You can check the version we return by going to `https://your.domsin.tld/api/config`

This does display the correct version of vaultwarden. Thanks for the link.

I guess now i'm not sure if its an issue but it doesn't display version at all for the extension about page.

**Reply by BlackDex** [reply]
I don't think we can fix that on the server side.
Clients are not managed by the Vaultwarden project.
